app/controllers/v0/claims_controller.rb

```diff
@@ -4,13 +4,18 @@ class ClaimsController < ApplicationController
     skip_before_action :authenticate
 
     def index
-      render json: current_user.claims,
+      render json: Claim.fetch_all(current_user.vaafi_headers),
```

Comment:
My end goal for this is to have the controller instantiate a client with the current user like RX is currently doing, so none of the models need to know about how to get a client or attributes from the user. I am not sure where we should do the translation from the API response to the model though and am open to any suggestions. I think @saneshark had some ideas on that as part of building a base client object and plugging in different parsers. It feels weird to have something in `/lib` instantiate models from `/app` but I'm not sure of a better place to do it.